### PR TITLE
PXC-2959: Disable optimistic_pa and modify related mtr tests

### PR DESCRIPTION
**Pull request by dutow**
According to support, this change will reduce data inconsistency issues
with workloads with foreign keys.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [submodule "percona-xtradb-cluster-galera"]
 	path = percona-xtradb-cluster-galera
-	url = https://github.com/percona/galera
-	branch = 4.x-8.0
+	url = https://github.com/dutow/galera
```
Comment (kamil-holubicki) [inline .gitmodules]: https://github.com/percona/galera

**Review by kamil-holubicki — CHANGES_REQUESTED** [review]
Could you please add message comment describing the issue solved here?

File: .gitmodules (hunk 1 above)
Comment (dutow) [reply .gitmodules]: I will update this after the galera commit is merged. I needed this change for jenkins testing

**Review by kamil-holubicki — APPROVED** [review]
lgtm

**Pull request merged**